src/mottakerroller.js

```diff
@@ -0,0 +1,27 @@
+const mottakerroller = [
+  {
+    kode: 'BRUKER_ELLER_BRUKERS_FULLMEKTIG',
+    term: 'Bruker eller brukers fullmektig'
+  },
+  {
+    kode: 'VIRKSOMHET',
+    term: 'Virksomheten saken er tilknyttet'
+  },
+  {
+    kode: 'ARBEIDSGIVER_ELLER_ARBEIDSGIVERS_FULLMEKTIG',
+    term: 'Arbeidsgiver eller arbeidsgivers fullmektig'
+  },
+  {
+    kode: 'ANNEN_ORGANISASJON',
+    term: 'Annen organisasjon'
+  },
+  {
+    kode: 'UTENLANDSK_TRYGDEMYNDIGHET',
+    term: 'Utenlandsk trygdemyndighet i avtaleland'
+  },
+  {
+    kode: 'NORSK_MYNDIGHET',
+    term: 'Andre offentlige etater'
+  },
```

Comment:
Synes term blir så rart med dette navnet :thinking: 
Men vi har vel ikke noe bedre, ettersom de ønsket "Andre offentlige etater" i beskrivelsen?

Reply:
Siden UTENLANDSK_TRYGDEMYNDIGHET ikke har noe til felles med NORSK_MYNDIGHET, kanskje vi kan bruke NORSK_ETAT?

Reply:
https://nav-it.slack.com/archives/C01CFP7SL4Q/p1675082007170799

Reply:
[2e673a2](https://github.com/navikt/melosys-kodeverk/pull/281/commits/2e673a20e1dc17f95ce424bfc365386965038416)